go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/golang/snappy v0.0.5-0.20231225225746-43d5d4cd4e0e
 	github.com/google/btree v1.0.1
 	github.com/google/pprof v0.0.0-20210827144239-02619b876842
-	github.com/google/uuid v1.5.0 // indirect
+	github.com/google/uuid v1.6.0 // indirect
```

Comment:
this is still core infra :(

Reply:
...indeed :) put up https://github.com/cockroachdb/cockroach/pull/135324 to upgrade that separately